tutor/templates/build/forum/bin/docker-entrypoint.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/sh -e
 
-export MONGOHQ_URL="mongodb://$MONGODB_AUTH$MONGODB_HOST:$MONGODB_PORT/cs_comments_service"
+if test "$MONGODB_PROTOCOL" = 'mongodb+srv'; then
+  export MONGOHQ_URL="$MONGODB_PROTOCOL://$MONGODB_AUTH$MONGODB_HOST/cs_comments_service$MONGODB_PARAMS"
+  export MONGO_WAIT=""
+else
+  export MONGOHQ_URL="$MONGODB_PROTOCOL://$MONGODB_AUTH$MONGODB_HOST:$MONGODB_PORT/cs_comments_service$MONGODB_PARAMS"
+  export MONGO_WAIT="-wait tcp://$MONGODB_HOST:$MONGODB_PORT"
+  echo "Waiting for mongodb..."
```

Comment:
I suggest you write: `services='mongodb/elasticsearch'` and then: `echo "Waiting for $services"`.